clang/lib/CodeGen/CGLoopInfo.h

```diff
@@ -111,6 +149,9 @@ class LoopInfo {
   /// Return this loop's access group or nullptr if it does not have one.
   llvm::MDNode *getAccessGroup() const { return AccGroup; }
 
+  // Recursively adds the metadata for this Array onto this GEP.
+  void AddIVDepMetadata(const ValueDecl *Array, llvm::Instruction *GEP) const;
```

Comment:
Sorry for not catching this earlier - the casing is still off. I've looked through the others carefully to avoid new comments